### PR TITLE
-All parameter now works with scope of Individual.

### PR DESCRIPTION
**Pull request by shpittam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,12 +65,18 @@ Connect-PowerBIServiceAccount   # or Login-PowerBIServiceAccount
...
 ### Get Workspaces
 
-Get all workspaces for the user:
+Get workspaces for the user. By default (without [-First](https://github.com/Microsoft/powerbi-powershell/blob/master/src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md#-first)) it returns the first 100 workspaces.:
 
 ```powershell
 Get-PowerBIWorkspace
 ```
 
+Introduced [-All](https://github.com/Microsoft/powerbi-powershell/blob/master/src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md#-all) parameter to retrieve all the workspaces for the user.
```
Comment (shpittam) [inline README.md]: This link would work once the code it merged into master.

Comment (CodeCyclone) [reply README.md]: we shouldn't hyperlink it because that will be invalid when this page gets merged to MicrosoftDocs.

File: README.md [hunk 2]
```diff
@@ -65,12 +65,18 @@ Connect-PowerBIServiceAccount   # or Login-PowerBIServiceAccount
 
 ### Get Workspaces
 
-Get all workspaces for the user:
+Get workspaces for the user. By default (without [-First](https://github.com/Microsoft/powerbi-powershell/blob/master/src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md#-first)) it returns the first 100 workspaces.:
```
Comment (CodeCyclone) [inline README.md]: Shows the first 100 workspaces assigned to the user.

File: README.md (hunk 1 above)
Comment (CodeCyclone) [inline README.md]: Shows all workspaces assigned to the user.

File: src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ Get-PowerBIWorkspace [-Scope <PowerBIUserScope>] [-Filter <String>] [-User <Stri
 
 ## DESCRIPTION
 Retrieves a list of Power BI workspaces that match the specified search criteria and scope.
+By default (without [-First](https://github.com/Microsoft/powerbi-powershell/blob/master/src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md#-first)) it returns the first 100 workspaces.
```
Comment (CodeCyclone) [inline src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md]: Don't put hyperlinks in here, I'm not sure how PlatyPS will treat them but this text gets emitted in Get-Help for PowerShell

Comment (CodeCyclone) [inline src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md]: Say: The first 100 workspaces are returned, use -First and -Skip to fetch more workspaces or use -All to return all workspaces

File: src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md [hunk 4]
```diff
@@ -111,7 +120,7 @@ Aliases: Top
 
 Required: False
 Position: Named
-Default value: 5000
+Default value: 100
```
Comment (CodeCyclone) [inline src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md]: Does this persist if you do another build again or does it change the default value back?

Comment (shpittam) [reply src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md]: It does persist upon multiple builds

**Pull request merged**